cmd/cluster-node-tuning-operator/main.go

```diff
@@ -67,7 +74,9 @@ func init() {
 
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(tunedv1.AddToScheme(scheme))
-	utilruntime.Must(mcov1.AddToScheme(scheme))
+	if !config.InHyperShift() {
```

Comment:
If there might be more additions to this clause I would bump it to the top or the bottom for visibility of future developments.